package.json

```diff
@@ -23,7 +23,6 @@
     "node": ">= 12"
   },
   "dependencies": {
-    "bluebird": "^3.7.2",
```

Comment:
I don't see any changes removing dependency on bluebird, are you sure it is safe to drop it?

Reply:
There was none, other than a single line: A cal to Promise.promisify, which just adds methods to the prototype. Since we are aiming for a new major version, and I'm going to rewrite all methods as async functions anyway, this does not matter.